README.md

```diff
@@ -12,7 +12,7 @@ languages in [SonarQube](http://www.sonarqube.org/), [SonarCloud](https://sonarc
 
 ## Features
 
-* [370+ C# rules](https://rules.sonarsource.com/csharp) and [160+ VB.&#8203;NET rules](https://rules.sonarsource.com/vbnet)
+* [370+ C# rules](https://rules.sonarsource.com/csharp) and [170+ VB.&#8203;NET rules](https://rules.sonarsource.com/vbnet)

```

Comment:
With this one we reach 171 on the VB.NET rules count. HasCorrectRuleCount_VB on the UT is checking the rules count reported on the README file through a regex to be inside the +10 range. That's why we need to bump it to 170+.